Sprintfinity3902/Entities/Items/AbstractItem.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Sprintfinity3902.Interfaces;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Sprintfinity3902.Entities.Items
+{
+    public abstract class AbstractItem : AbstractEntity,  IItem
```

Comment:
I reallllly like this class, I think I want to see something similar for blocks, enemies, etc.